lib/mutant/config.rb

```diff
@@ -60,6 +35,32 @@ class Config
       mutant.yml
     ].freeze
 
+    private_constant(*constants(false))
+
+    class CoverageCriteria
+      include Anima.new(:timeout, :test_result)
```

Comment:
@dgollahon The next step is to allow to configure the coverage criteria *per subject* in the config file an in-line.

Same for the mutation timeout.

The ideal setting than is a global "high" timeout with `coverage_criteria.timeout = false` and a per subject one after manual review (From an alive, example an infinite loop) setting it to true.

Reply:
Also there are more possible criteria like `warnings` etc.

Reply:
And the most important criteria for @pawelpacana counting ruby aborts as killed.

https://github.com/mbj/mutant/issues/1054.

Next major feature for 0.10.8. The infrastructure exists now.

Per subject config will be added in 0.10.9 possibly also in-line configuration.